ml/models/logo_detection/data.py

```diff
@@ -0,0 +1,11 @@
+class BoundingBox:
+    def __init__(self, x: float, y: float, w: float, h: float):
+        self.x = x
+        self.y = y
+        self.w = w
+        self.h = h
+
+class LogoDetectionInference:
+    def __init__(self, boundingBox: BoundingBox, confidence: float):
+        self.boundingBox = boundingBox
```

Comment:
Python styling suggests the use of `snake_case` for variables instead of `camelCase`
```suggestion
class LogoDetectionInference:
    def __init__(self, bounding_box: BoundingBox, confidence: float):
        self.bounding_box = bounding_box
```